src/org/usfirst/frc/team1024/robot/Robot.java

```diff
@@ -39,6 +43,8 @@ public void robotInit() {
 		m_chooser.addDefault("Default Auto", new Auto());
```

Comment:
What is m_chooser?

Reply:
It's declared above as a SendableChooser<Command>